### PR TITLE
Removes now-unnecessary hotfix, bumps compiler version

### PR DESCRIPTION
**Pull request by ike709**
Fixed in 513.1534

### CODE REVIEW

**Comment by Blueturbo47** [comment]
mmm, cookie

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by ike709** [comment]
@PowerfulBacon Merge it you fool the fix was terrible code.

File: code/_compile_options.dm [hunk 1]
```diff
@@ -33,11 +33,11 @@
 
 //Update this whenever you need to take advantage of more recent byond features
 #define MIN_COMPILER_VERSION 513
-#define MIN_COMPILER_BUILD 1514
+#define MIN_COMPILER_BUILD 1534
```
Comment (ike709) [inline code/_compile_options.dm]:
```suggestion
#define MIN_COMPILER_BUILD 1514
```

File: code/_compile_options.dm [hunk 2]
```diff
@@ -37,7 +37,7 @@
 #if DM_VERSION < MIN_COMPILER_VERSION || DM_BUILD < MIN_COMPILER_BUILD
 //Don't forget to update this part
 #error Your version of BYOND is too out-of-date to compile this project. Go to https://secure.byond.com/download and update.
-#error You need version 513.1514 or higher
+#error You need version 513.1534 or higher
```
Comment (ike709) [inline code/_compile_options.dm]:
```suggestion
#error You need version 513.1514 or higher
```

**Comment by ike709** [comment]
No longer necessary, TGUI4 PR dealt with it.

**Pull request closed without merging**